streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -544,6 +563,11 @@
 
             // MEDIUM
 
+            .define(ASSIGNMENT_ACCEPTABLE_RECOVERY_LAG_CONFIG,
+                    Type.LONG,
+                    1_000L,
```

Comment:
@vvcephei suggested this be `10,000` which seemed high to me. But I'm willing to be convinced 🙂

Reply:
Looking at the stateheavy benchmark that we run, 10000 does not seem too unreasonable. In the benchmarks 10 state stores are filled and the benchmark reports a throughput of around 7000 records per second. But I am also fine with 5000 as default. 1000 seems a bit low to me.

Reply:
Alright, we can go with 10,000 for the default